lib/spdy/server.js

```diff
@@ -237,6 +237,9 @@ proto.emit = function emit(event, req, res) {
   res.writeContinue = spdy.response.writeContinue;
   res.spdyStream = handle.getStream();
 
+  req.res = res;
```

Comment:
These are exported to the user. What do you think about putting them as `req._req`? Also, `req.res` doesn't seem to be used anywhere, let's not export it ;)